backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -203,14 +300,100 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
             }
         }
 
+        /// <summary>
+        /// Append a new note to the project.
+        /// Parse the 'noteType' value, or set it as 'General'.
+        /// </summary>
+        /// <param name="project"></param>
+        /// <param name="noteType"></param>
+        /// <param name="note"></param>
+        private void AddNote(Entity.Project project, string noteType, string note)
+        {
+            if (!String.IsNullOrWhiteSpace(note))
+            {
+                if (!Enum.TryParse(noteType, true, out Entity.NoteTypes type))
+                {
+                    type = noteType switch
+                    {
```

Comment:
Should these checks ignore caps?

Reply:
Probably, but similar to other implementation decisions, I've taken a few short-cuts.  This helper is currently only to support imports.  Which at this point in time are only for development and one time run in production.  I do try to make them as generic and future proof as I can within the timeframe.